### ISSUE TITLE
ogg Radio Stream stops after every Title

### ISSUE THREAD

**Issue by avantgardex** [clementine-player/Clementine#5114]
after update to 1.2.3 ogg Stream stops after every song. Like this one http://ogg.theradio.cc/
mp3 stream works fine.

I am using Linux Mint 17.1 cinamon

**Comment by chiak597** [comment]
I have the same problem with this stream: http://stream.funradio.sk:8000/fun128.ogg (mp3 version works fine).

To be more precise, playback stops after each change of ogg stream metadata.

**Comment by Chocobozzz** [comment]
Can someone test with the latest builds? http://builds.clementine-player.org/

I can't reproduce with the stream of @avantgardex

**Comment by narunlifescience** [comment]
unable to reproduce here too..

**Comment by chiak597** [comment]
I played second stream (FunRadio) with freshly compiled git master (4db358a). Unfortunately it still doesn't work correctly. Playback stops for couple of seconds (5-15) after each change of metadata. After that time, playback continues pretty normally.

I removed ~/.config/Clementine with no luck.

**Comment by narunlifescience** [comment]
I also experienced a small delay when song changes but the stream never stopped & continued to play ...

**Comment by chiak597** [comment]
Well, probably I didn't describe my problem correctly. It's not about complete stream stop, but rather about the delay you experienced. 

This delay is really annoying. Only OGG streams are affected by this, MP3 streams are working normally.

**Comment by WildPenquin** [comment]
I have a very long delay (I used to think it is a complete stop) at every change of metadata (song change) in streams at kohina.com. 

Perhaps 0.2-0.5s is played when metadata changes (sometimes), then only silence is produced (althouth the Clementine UI shows as if the stream is still playing). After a minute or so (could be the **next** metadata change) playback resumes. After a while, clementine goes on to the next item on the playlist.

So seems that clementine is utterly confused about metadata change and partly thinkgs the stream has ended? This used to work previously (not sure when since I don't listen to the streams regularly), which means this is a regression (OTOH, it is certainly possible they did something to the metadata at kohina.com, but previously I saw the titles changing as they should).

This is on Arch Linux, 1.3.1.r228.gafe34ee-1 compiled from AUR (I can also reproduce on 1.3.1-2 which is in the official Arch repositories).

**Comment by maakuth** [comment]
Reproduces here with Kohina.com (stream url https://kohina.duckdns.org/icecast/stream.ogg) using  Ubuntu's Clementine package: 1.3.1+git276-g3485bbe43+dfsg-1.1build1

**Comment by MrManor** [comment]
Another Radioparadise lover here. I see the same behavior on Version 1.3.1 on Opensuse. This forces me to choose aac streams, as RP is my primary music source. [96k Ogg Vorbis (with song title data)](http://icy-7.radioparadise.com/rp_96m.ogg)
According to KDE settings I am using phonon-gstreamer-4.9.0

**Comment by JulianVolodia** [comment]
Hi @MrManor 

Thanks for info. Could you provide any log about this behaviour on your configuration?

Could you test against bleeding-edge version from master?

Best!

**Comment by MrManor** [comment]
Hi @JulianVolodia 
> Thanks for info. Could you provide any log about this behaviour on your configuration?
[clmlog.zip](https://github.com/clementine-player/Clementine/files/3613130/clmlog.zip)
Just after 22:09 first there is a short pause and a few seconds later music stops completely.  Just at the same time as the song title changes. 

> Could you test against bleeding-edge version from master?
I am afraid this is above my competence. I always install from rpm. And as this computer is my daily workhorse I only install from stable repositories on it. My other PC's is currently on storage as we are moving apartment. 

Regards
Klaus

**Comment by JulianVolodia** [comment]
@MrManor great thanks! :) I return to you asap.

**Comment by JulianVolodia** [comment]
Dear @MrManor 

could you please test behaviour on your side.
1) start listening to ogg stream
2) after it 'stops' (actually not stopped, there is a Pause icon in place of Play icon) left it untouched

After few minutes (less than 10 minutes here) Clementine start to stream again.

Tell me if you seeing this behaviour also. Cheers.

**Comment by MrManor** [comment]
Thanks for looking into this. It seems to bee a general problem, not only for Clementine: On the [stream link page](https://radioparadise.com/listen/stream-links) of Radioparadise I found: "_Stream metadata (song title display in the player) in Ogg streams causes glitches in some players (notably VLC & certain hardware units). That's why we provide Ogg streams both with and without titles._" 
I would bet that the mentioned hardware players use some embedded Linux :-)

[clm-log.zip](https://github.com/clementine-player/Clementine/files/3617692/clm-log.zip)

Yes I actually see the behavior described. It is a freeze, not a stop. In the attached log I let Clementine play over a couple of numbers.  At the end of the first number first a very short break, and then a few seconds later Clementine freezes, visualizer also freezes with some bars still up. 

This time music resumes after some seconds with the second number.

As the second mucic number nears the end the short freeze/break again - and shortly after a complete freeze for something like 10 minutes. I suspect until the fourth number starts, but that might be coincidence. 
This time It only plays for a short while, - then gives up and enters Stop (pause buttom becomes a play buttom)

**Comment by JulianVolodia** [comment]
Yup. Last, if you have any next entry in playlist it will go next to it.

**Comment by axelsimon** [comment]
If this is an upstream problem, could we figure out in which upstream project and help fix it there?
If it's specific to Clementine (even if the *symptom* is shared with other players), can we try and get it solved here?

It's been close to 5 years, surely we can do something?

**Comment by cyraid** [comment]
Mine also doesn't play ogg's from Rainwave anymore, just mp3. Says:
> Your GStreamer Installation is missing a plug-in
